bolt/include/bolt/Core/BinaryContext.h

```diff
@@ -670,6 +670,9 @@ class BinaryContext {
   /// Indicates if the binary is Linux kernel.
   bool IsLinuxKernel{false};
 
+  /// Linux kernel version (major, minor, reversion)
+  std::tuple<unsigned, unsigned, unsigned> LinuxKernelVersion;
```

Comment:
This maybe should be a struct instead of a tuple

Reply:
Done